parser-typechecker/src/Unison/Codebase/Editor/Git.hs

```diff
@@ -48,10 +60,11 @@ pullBranch localPath uri treeish = do
   case e of
     Left e -> throwError (SomeOtherError (Text.pack (show e)))
     Right _ -> pure ()
-  "git" (["clone", "--quiet"] ++ ["--depth", "1"]
-     ++ maybe [] (\t -> ["--branch", t]) treeish
-     ++ [uri, Text.pack localPath])
-    `onError` throwError (NoRemoteRepoAt uri)
+  withStatus ("Downloading from " ++ Text.unpack uri ++ " ...")
+    ("git" $^ (["clone", "--quiet"] ++ ["--depth", "1"]
+       ++ maybe [] (\t -> ["--branch", t]) treeish
+       ++ [uri, Text.pack localPath]))
+      `onError` throwError (NoRemoteRepoAt uri)
```

Comment:
The `withStatus` line was added, and the rest was indented into it.
The `$^` was inserted between "git" and "clone" to suppress printing the command.